source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,90 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.7
+----------------------------
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Release day TBD**
+
+Breaking Changes or Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+- Added support for BrowserView.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+Windows
+^^^^^^^^^^^^^
+- Removed the dark mode toggle option for Windows.
+
+MacOS
+^^^^^^^^^^^^^
+-
+
+All Platforms
+^^^^^^^^^^^^^
+- Added Swedish language to the spellchecker.
+- Added a setting to specify the default desktop app download location.
+- Updated the desktop app loading indicator.
+- Restored deeplinking.
+- Switched the dictionary to the one Electron uses to have the ability to fix spelling.
+- Added improvements to be consistent on the use of URL and URL libraries.
+- Removed the Finder functionality from the Desktop App.
+- Updated CSP for Desktop App to avoid warnings in the dev tools.
+- Removed the menu bar in the Settings window for Windows and Linux.
+- Shortened the max-length/width for server tab names to 224px.
+- For Windows and Linux, the system language is now used and the spell checker's option help text was improved. languages are no longer selectable and toggling the spellchecker requires a restart.
+- Udpated the menubar and system tray icons for improved contrast.
+- Removed ``libappnotify1`` as a dependency requirement in Debian installers as it is no longer shipped in Debian's Bullseye. It is still recommeded to install where available.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron to v11.4.4. Electron is the underlying technology used to build the Desktop apps.
+- Added support for M1 architecture in the build pipeline.
+
+Bug Fixes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Windows
+^^^^^^^^^^^^^
+- Fixed an issue where Windows desktop notifications did not auto-dismiss when another notification arrived.
+- Fixed an issue on Windows where the **Pin to Taskbar** icon got lost during an upgrade.
+
```

Comment:
I'll have one note to add from this PR: https://github.com/mattermost/desktop/pull/1590